app/Models/User.php

```diff
@@ -2,44 +2,107 @@
 
 namespace App\Models;
 
-// use Illuminate\Contracts\Auth\MustVerifyEmail;
-use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 use Laravel\Sanctum\HasApiTokens;
 
 class User extends Authenticatable
 {
-    use HasApiTokens, HasFactory, Notifiable;
+    use HasApiTokens, Notifiable;
 
     /**
-     * The attributes that are mass assignable.
-     *
-     * @var array<int, string>
+     * User Attributes
+
+     * $this->attributes['id'] - int - contains the user primary key (id).
+     * $this->attributes['name'] - string - contains the user's name.
+     * $this->attributes['password'] - string - contains the password set by user.
+     * $this->attributes['address'] - string - contains the user's address.
+     * $this->attributes['balance'] - int - contains the user's balance (available and spendable money).
```

Comment:
Debe ser de tipo float (en el diagrama no está así pero por facilidad de operacion)
```suggestion
     * $this->attributes['balance'] - float - contains the user's balance (available and spendable money).
```